assembly/fabric8-ide-dashboard-war/pom.xml

```diff
@@ -254,22 +288,21 @@
                                 </configuration>
                             </execution>
                             <!--   3.5 Run npm tests locally                                -->
-                            <execution>
-                                <id>compilation</id>
-                                <phase>test</phase>
-                                <goals>
-                                    <goal>run</goal>
-                                </goals>
-                                <configuration>
-                                    <target unless="skipTests">
+                            <!-- <execution> -->
```

Comment:
Block comment would make diff way simpler. Are you going to merge with these lines commented or is it for the testing purposes?

Reply:
I don't really know how this even happened. Fixed.